app.py

```diff
@@ -1,31 +1,33 @@
 from flask import Flask, render_template, request, redirect, url_for
-import session_items as session
+from config import TRELLO_KEY
+import trello
+from status import COMPLETED
 
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
```

Comment:
If you're not using flask session cookies then flask_config.py is now redundant (along with the SECRET_KEY)